src/Enhavo/Bundle/AppBundle/Resources/translations/EnhavoAppBundle.de.yml

```diff
@@ -33,5 +33,18 @@ tab:
     seo: 'SEO'
 table:
   empty: 'Keine Daten vorhanden'
+  batch:
```

Comment:
If you use prefixes you can also put them in sub spaces. You should also categorize them like in this [guide](http://obtao.com/blog/2013/06/how-to-organize-your-translations-in-symfony)

My suggestion would be:

```yml
action:
  submit: 'Ausführen'
  delete: 'Löschen'
  close: 'Schließen'
message:
  confirm:
    generic: 'Wollen sie wirklich das Kommando %command% für alle ausgewählten Elemente ausführen?'
    yes: 'Ja'
    no: 'Nein'
    delete: 'Wollen sie wirklich alle ausgewählten Elemente löschen?'
error:
  noneSelected: 'Keine Elemente ausgewählt'
  noActionSelected: 'Keine Aktion ausgewählt'
label:
  action: 'Aktion auswählen'
```